admin_guide/install/advanced_install.adoc

```diff
@@ -97,16 +75,56 @@ Removing this line til we have a better instruction:
 # git checkout -b 3.x v3.0.0
 ////
 
+[[configuring-ansible]]
+
 == Configuring Ansible
 
-You must create an *_/etc/ansible/hosts_* file for the playbook to use during
-the installation. The following is an example of a Bring Your Own (BYO) host
-inventory, based on the link:#default-hosts[example host environment]. You can
-see these example hosts present in both the *[masters]* and *[nodes]* sections:
+You must create an *_/etc/ansible/hosts_* file, which is Ansible's inventory
+file for the playbook to use during the installation.
+
+Any hosts you designate as masters during the installation process should also
+be configured as nodes that are marked
+link:../../admin_guide/manage_nodes.html#marking-nodes-as-unschedulable-or-schedulable[unschedulable].
+This is so the masters are configured as part of the
+link:../../architecture/additional_concepts/networking.html#openshift-sdn[OpenShift
+SDN].
+
+The following sections describe example Bring Your Own (BYO) host inventory
+files for various environment topographies, including configuring
+link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#high-availability-masters[multiple
+masters for high availability]. You can choose an example that matches your
```

Comment:
@adellape The sentence "The following sections ..." is difficult (for me) to read.  I think the core structure is "describe NP for ..." where the primary N in the NP is "files", and everything else is a piling-on of adjectives (made from other nouns) that decorate "files".
What kind of files?
A: inventory files.
What kind of inventory files?
A: host inventory files
What kind of host inventory files?
A: BYO host inventory files?
What kind of BYO host inventory files?
A: example BYO host inventory files
I think there is actually a style-guide point about this kind of chain, but am pre-coffee atm (which could explain also any mis-parsing going on)...